README.md

```diff
@@ -75,16 +77,17 @@ GCC 4.9 / Clang 3.2 or later is recommended because OpenSpades relies on C++11 f
    cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo && make
    ```
 
-4. Install OpenSpades (optional but recommended):
-
-   `sudo make install`
-
-   **note**: If you have a previous installation of OpenSpades, you have to uninstall it manually by `sudo rm -rf /usr/local/share/games/openspades` before installing a new one.
-
-5. Launch:
-
-   `openspades` (if installed) or `cd $REPO_DIRECTORY/openspades.mk; bin/OpenSpades` and enjoy
-
+4. Launching the game (follow the steps either A or B)
+    4.A Install OpenSpades (recommended):
+       `sudo make install`
+       **note**: If you have a previous installation of OpenSpades, you have to uninstall it manually by `sudo rm -rf /usr/local/share/games/openspades` before installing a new one.
+    4.B Portable
+        Copy the Resources directory into bin (or else the game won't launch):
+        ```cp -r ./Resources ./bin/```
+        If you plan on distributing it, remember to remove CMake files and folders from Resources.
+        Move the "/openspades.mk" folder somewhere else, for example `/home/user/Games`, or `/opt/games` and rename it to "/OpenSpades". Game launcher is in `~/bin/OpenSpades`.
```

Comment:
The Linux version doesn't really support a "portable" mode. It just happens to pick up `Resources` from a current working directory to obviate the need to reinstall on every build during development.

(The Windows version, on the other hand, includes a special logic to enable a portable mode.)

Reply:
I'm confused ':-D, doesn't this describe how every portable app works? I'm using Waterfox as a portable, and it uses subfolders inside its directory as well. I'm just happy that I can .zip built OpenSpades game and unpack on another computer and it'll work

Reply:
<https://en.wikipedia.org/wiki/Portable_application>:

> A portable application (portable app) [...] is a [program](https://en.wikipedia.org/wiki/Computer_program) designed to read and write its configuration settings into an accessible folder in the computer, usually in the folder where the portable application can be found.

... which the Linux version of OpenSpades is not designed to do.

<https://old.reddit.com/r/waterfox/comments/bmit86/bizarre_message_from_waterfox_wizard/en6vjss/?context=3>:

> So the appellation "Portable Waterfox" is misleading as it is not really portable at all (even if it does not leave traces in the registry, *it does leave traces on your computer*, which a portable application should never do), and it will aways have that additional profile folder in the /AppData/local/Waterfox directory.

The Linux version of OpenSpades does leave traces by [storing](https://github.com/yvt/openspades/blob/e6d1856b3d8c937392883d2fb43c43175df4610b/Sources/Gui/Main.cpp#L396-L444) configuration files in a home directory.

Reply:
Hmm, "(..) designed to read and write its configuration settings into an accessible folder in the computer" -> "(Portable Waterfox) is misleading as it is not really portable (..) profile folder in the `(..)/AppData/local/Waterfox`"
`/AppData/local` in Windows is an accessible folder, separate for each user on the computer and a common directory for app settings, like OpenSpades uses `/home/user/.local/share/openspades/` on Linux. Following the logic of the guy from Reddit post, there would be no apps defined by Portable because most of apps that don't require installation tend to save user preferences in user-specific directories.

As for OpenSapdes, I renamed this step to "Copying OpenSpades manually" in the Readme. I never used GitHub for things other than downloading, I'm not sure if I'm using the editor properly. I think I committed changes and submitted it for review.

Reply:
I'm pretty sure Wikipedia's definition used the word "accessible" in a figurative or non-strict sense, since if a program's configuration were literally inaccessible by the user, it would be inaccessible by the program as well. If you read it further, the Wikipedia page gives a clearer example of an "accessible" folder: "the folder where the portable application can be found". This is the definition I've known for a decade and used by a number of applications, including [KeePassXC](https://github.com/keepassxreboot/keepassxc/blob/2b8d670f17e22722b44c1718a02322893de43418/src/core/Config.cpp#L482-L486), [Sublime Text](https://github.com/sublimehq/sublime_text/issues/3786#issuecomment-733330912), and [OpenSpades](https://github.com/yvt/openspades/blob/e6d1856b3d8c937392883d2fb43c43175df4610b/Sources/Gui/Main.cpp#L341-L342) (Windows build only).

> As for OpenSapdes, I renamed this step to "Copying OpenSpades manually" in the Readme.

Fair enough.

> I never used GitHub for things other than downloading, I'm not sure if I'm using the editor properly. I think I committed changes and submitted it for review.

You are doing great. Though, you should use the preview tab to check the Markdown rendering because the formatting is still broken (see https://github.com/yvt/openspades/pull/1011#discussion_r861445626).

Reply:
Alright, it makes sense. As a note about Waterfox it allows specifying User Profile directory, so it can be set to pendrive with the installation, making it truly portable, but I don't think it would be reliable on Windows if pendrive's mount letter would change. I made changes to formatting, hope you'll like it!